lib/ui/components/cv_stateful_button.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class CVStateFulButton extends StatefulWidget {
```

Comment:
Can we choose a better indicative name here ?

Reply:
Suggest please, I can't find any good name as I haven't build the complete app. 😅

Reply:
You can call it `CVFlatButton`

Reply:
@Nitish145 @manjotsidhu Done in latest commits.